Formula/grass.rb

```diff
@@ -1,7 +1,7 @@
 class Grass < Formula
   desc "Geographic Resources Analysis Support System"
   homepage "https://grass.osgeo.org/"
-  revision 1
+  revision 2
```

Comment:
This  formula is  missing a `test do` block. It appears to be a GUI tool (built from source) and if adding a `test do` block is even feasible, I won't be able to do it.